data/crsjson.schema.json

```diff
@@ -0,0 +1,935 @@
+{
+  "$id": "https://proj.org/crsjson.schema.json",
```

Comment:
Now that you have decided to go with the name PROJJSON, shouldn't it then be `projjson.schema.json`?